### PR TITLE
[DOCS] Adding link from SAML docs to ADFS blog

### LINKED ISSUE #61668: [DOCS] Link in Elastic documentation of SAML to  Blogpost on elasticsearch-saml-authentication-with-adfs
For SAML single-signing for Elastic Stack the request is to add within our general elastic documentation for SAML with ADFS (e.g.: https://www.elastic.co/guide/en/elasticsearch/reference/6.8/security-troubleshooting.html ) a link to the following blogs:

https://www.elastic.co/blog/how-to-configure-elasticsearch-saml-authentication-with-adfs 

Since this blog goes into detail on troubleshooting I think it would be beneficial to refer to this our documentation.

### PR DESCRIPTION
**Pull request by lockewritesdocs**
Adds a link to the blog for [configuring Elasticsearch SAML with ADFS](https://www.elastic.co/blog/how-to-configure-elasticsearch-saml-authentication-with-adfs) from the identity provider page. 

Closes #61668

### CODE REVIEW

**Comment by lockewritesdocs** [comment]
@InbarShimshon, please let me know if this change satisfies the issue you raised in #61668.

**Comment by InbarShimshon** [comment]
Hi @lockewritesdocs I checked the https://www.elastic.co/guide/en/elasticsearch/reference/current/saml-guide-idp.html and didnt see a clear link here, or did you refer to a different page in our doc? Thank you for helping on this one!

**Comment by lockewritesdocs** [comment]
@InbarShimshon, apologies for not adding the link to [the preview server](https://elasticsearch_62006.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/saml-guide-idp.html) for this PR. That page shows the change, which includes a link to the ADFS blog. 

image

**Comment by InbarShimshon** [comment]
Ah! I see :) awesome, yes looks good to me :) Thank you @lockewritesdocs

File: x-pack/docs/en/security/authentication/saml-guide.asciidoc [hunk 1]
```diff
@@ -28,7 +28,8 @@ to login. The <<saml-kibana>> section provides more detail about how this works.
 The Elastic Stack supports the SAML 2.0 _Web Browser SSO_ and the SAML
 2.0 _Single Logout_ profiles and can integrate with any Identity Provider (IdP)
 that supports at least the SAML 2.0 _Web Browser SSO Profile_.
-It has been tested with a number of popular IdP implementations.
+It has been tested with a number of popular IdP implementations, such as
+https://www.elastic.co/blog/how-to-configure-elasticsearch-saml-authentication-with-adfs[Microsoft Active Directory Federation Services (ADFS)].
```
Comment (InbarShimshon) [inline x-pack/docs/en/security/authentication/saml-guide.asciidoc:32]: looks good! thank you

**Review by InbarShimshon — APPROVED** [review]

**Review by InbarShimshon — APPROVED** [review]

**Pull request merged**